.github/workflows/ci-e2e-checks.yaml

```diff
@@ -9,25 +9,25 @@ on:
       - master
 
 jobs:
- 
+
   e2e-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Is there a specific reason you pinned this to 18.04? I don't think we want to pin this right now.

Reply:
No, I forgot it there from my tests.

Reply:
It looks like this is still pinned to ubuntu-18.04. Looking back through your latest commits it looks like you changed it back...then reverted the change in a later commit.